mvrx-mocking/src/main/kotlin/com/airbnb/mvrx/mocking/MockableMavericks.kt

```diff
@@ -109,22 +117,64 @@ object MockableMavericks {
      * system is automatically enabled.
      *
      * Calling this subsequent times will replace the plugins with new instances.
-     *
-     * @param debugMode True if debug checks should be enabled
-     * @param mocksEnabled True if ViewModel mocking should be enabled.
-     * @param context Application context. If provided this will be used to register a
-     * [ViewModelStatePrinter] for each ViewModel to support mock state printing.
      */
-    fun initialize(mocksEnabled: Boolean, debugMode: Boolean, context: Context?) {
+    fun initialize(
+        /**
+         * True if ViewModel mocking should be enabled.
+         */
+        mocksEnabled: Boolean,
+        /**
+         * True if debug checks should be enabled.
+         */
+        debugMode: Boolean,
+        /**
+         * Application context. If provided this will be used to register a [ViewModelStatePrinter] for each ViewModel to support mock state printing.
+         */
+        applicationContext: Context?,
+        /**
+         * Provide a default coroutine context for viewModelScope in all view models. It will be added after [SupervisorJob]
+         * and [Dispatchers.Main.immediate].
+         */
+        viewModelCoroutineContext: CoroutineContext = EmptyCoroutineContext,
+        /**
+         * Provide a coroutine context that will be used in the [CoroutinesStateStore]. All withState/setState calls will be executed in this context.
```

Comment:
Can you indicate what the defaults are here? It looks like it defaults to EmptyCoroutineContext but in reality, it has a sane default but this just lets you override it.

Reply:
sure